### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by edshadi** [langfuse/langfuse#2489]
### Describe the bug

I am integrating Langfuse with Langchain (Typescript). The output in production is always null.
<img width="432" alt="Screenshot 2024-06-30 at 11 56 10 AM" src="https://github.com/langfuse/langfuse/assets/207551/7b677407-9d81-437c-9b89-9b5ce7181865">

### To reproduce

It's very simple langchain.js integration:

```
import { CallbackHandler } from 'langfuse-langchain';
const langFuseHandler = new CallbackHandler();

const model = new ChatOpenAI({
      modelName: process.env.OPENAI_API_MODEL,
      temperature: 0.2,
      metadata: {
        chatId
      },
      callbacks: [langFuseHandler],
    });
    
    const stream = await ragChain
      .withConfig({
        metadata: {
          chatId
        },
      })
      .stream(
        {
          input: currentMessageContent,
          chat_history: formattedPreviousMessages,
        },
        {
          callbacks: [langFuseHandler],
        },
      );
    
```

### SDK and container versions

"langfuse-langchain": "^3.11.2"

### Additional information

locally, the output comes through if I don't move away from the page. On production (deployed on Vercel), it doesn't capture the output at all.

### Are you interested to contribute a fix for this bug?

No

**Comment by marcklingen** [comment]
Have a look here: https://langfuse.com/docs/sdk/typescript/guide#lambda

tldr: in serverless environments you need to wait for the events to be sent to Langfuse. On Vercel, you can use waitUntil() to return the response and then wait for the events to be sent to langfuse.
